Cargo.toml

```diff
@@ -13,7 +13,7 @@ authors = ["Aurora Labs <hello@aurora.dev>"]
 # An update of the MSRV requires updating:
 # - `rust-toolchain` files in `near-plugins-derive/tests/contracts/**`
 # - the toolchain installed in CI via the `toolchain` parameter of `actions-rs/toolchain@v1`
-rust-version = "1.74.0"
+rust-version = "1.76.0"
```

Comment:
Is the MSRV bump required due to dependency updates?

Reply:
Unfortunately yes. The `near-workspaces 0.11` required 1.76.0.

Reply:
Heads up for the Bridge team that the MSRV is bumped here, cc @karim-en.